must-gather/collection-scripts/gather_namespaced_resources

```diff
@@ -67,37 +66,41 @@ oc_yamls+=("hpa")
 oc_yamls+=("subscription")
 oc_yamls+=("installplan")
 
-echo "collecting dump of namespace" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
-oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect ns/"${INSTALL_NAMESPACE}" --"${SINCE_TIME}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
-echo "collecting dump of clusterresourceversion" | tee -a  "${BASE_COLLECTION_PATH}"/gather-debug.log
-for oc_yaml in "${oc_yamls[@]}"; do
-   # shellcheck disable=SC2129
-   oc adm --dest-dir="${BASE_COLLECTION_PATH}" inspect "${oc_yaml}" -n "${INSTALL_NAMESPACE}" --"${SINCE_TIME}" >> "${BASE_COLLECTION_PATH}"/gather-debug.log 2>&1
-done
+echo "collecting existing namespaces" | tee -a "${BASE_COLLECTION_PATH}"/gather-debug.log
+INSTALL_NAMESPACES=$(oc get storagecluster -A --no-headers | awk '{print $1}')
```

Comment:
Is there a different way to find the required namespace ? In case the logs are collected for an issue where the storagecluster is not at all created.